ClosedXML_Tests/Excel/Saving/SavingTests.cs

```diff
@@ -1,15 +1,28 @@
 using ClosedXML.Excel;
 using NUnit.Framework;
+using System;
+using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Threading;
+using System.Threading.Tasks;
 
 namespace ClosedXML_Tests.Excel.Saving
 {
     [TestFixture]
     public class SavingTests
     {
+        private string _tempFolder;
+        private List<string> _tempFiles;
+
+        [SetUp]
+        public void Setup()
```

Comment:
I missed this. Can you check whether you can rather use the `using (var tf = new TemporaryFile(...))` pattern as in `OleDbTests`. Using that pattern would clean up any test files automatically.